### ISSUE TITLE
AUR packages?

### ISSUE THREAD

**Issue by DocMAX** [isjerryxiao/pacroller#7]
Does it take care of AUR packages, too?

**Comment by isjerryxiao** [comment]
Nope. You should always inspect the PKGBUILD carefully before installing or upgrading an aur package.
